lib/tasks/search.rake

```diff
@@ -3,55 +3,80 @@
 namespace :search do
   BATCH_SIZE = 1000
 
-  desc 'Reindex tags'
+  desc "Recreate tag index"
   task(index_tags: :environment) do
     TagIndexer.index_all
   end
-  desc 'Reindex pseuds'
+
+  desc "Recreate pseud index"
   task(index_pseuds: :environment) do
     PseudIndexer.index_all
   end
-  desc 'Reindex works'
+
+  desc "Recreate work index"
   task(index_works: :environment) do
     WorkIndexer.index_all
+    WorkCreatorIndexer.index_from_db
   end
-  desc 'Reindex bookmarks'
+
+  desc "Recreate boomark index"
   task(index_bookmarks: :environment) do
     BookmarkIndexer.index_all
   end
-  desc 'Reindex all'
+
+  desc "Reindex all works without recreating the index"
+  task(reindex_works: :environment) do
+    WorkIndexer.index_from_db
+    WorkCreatorIndexer.index_from_db
+  end
+
+  desc "Reindex all bookmarkables without recreating the index"
+  task(reindex_bookmarkables: :environment) do
+    BookmarkedExternalWorkIndexer.index_from_db
```

Comment:
Should we add a task specifically for this issue, which requires reindexing works and series but not external works?

Reply:
I can if you think it'd be a good idea, though I expect that the extra cost of reindexing all external works is probably pretty small relative to the cost of reindexing all works.

Reply:
Makes sense, yeah. Let's just use this task.